App/Components/ContactListItem/index.js

```diff
@@ -1,12 +1,13 @@
-import React from 'react';
-import {View, Text, Platform, TouchableOpacity, Linking} from 'react-native';
+import React, { useState } from 'react';
+import {View, Text, Platform, TouchableOpacity, Linking, Modal} from 'react-native';
```

Comment:
@anfezuar the react-native modal is deprecated. I recommend use https://github.com/react-native-community/react-native-modal